pydantic/main.py

```diff
@@ -19,6 +19,7 @@
 from .config import BaseConfig, Extra, build_config, inherit_config
 from .errors import PydanticUserError
 from .fields import Field, FieldInfo, ModelPrivateAttr
+from .generics_utils import create_generic_submodel, is_generic_model
```

Comment:
to avoid private methods being publicly importable (e.g. with `from pydantic.main import create_generic_submodel`, better to add `_generics` to the `from ._internal import ...` line.